packages/react-component-library/src/primitives/Container/partials/StyledContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components'
+import { selectors } from '@royalnavy/design-tokens'
+
+import { ContainerSizeType } from '../Container'
+import { CONTAINER_CONTENT_WIDTH } from '../constants'
+
+const { spacing } = selectors
+
+interface StyledContainerProps {
+  $size?: ContainerSizeType
+}
+
+export const StyledContainer = styled.div<StyledContainerProps>`
+  max-width: ${CONTAINER_CONTENT_WIDTH};
+  margin-left: auto;
+  margin-right: auto;
+
+  ${({ $size }) => {
+    if ($size === 'small') {
```

Comment:
It could be nice to reuse a `type` here and then the code could be more declarative.